aries/misc/test_utils/src/devsetup.rs

```diff
@@ -192,8 +192,14 @@ pub async fn dev_build_featured_anoncreds() -> impl BaseAnonCreds {
         return IndyCredxAnonCreds;
```

Comment:
We are using `--all-features` flag in CI, and I am using it myself in my current LSP. Although [this should be avoided if at all possible](https://doc.rust-lang.org/cargo/reference/features.html?highlight=additive#mutually-exclusive-features), I can still change both (and switch feature flags manually), I think for now, following a priority order of `credx`, `anoncreds-rs`, `mock` in case of "conflicting" feature flags is fine.